deltachat-ffi/deltachat.h

```diff
@@ -403,11 +403,8 @@ char*           dc_get_blobdir               (const dc_context_t* context);
  * - `send_port`    = SMTP-port, guessed if left out
  * - `send_security`= SMTP-socket, one of @ref DC_SOCKET, defaults to #DC_SOCKET_AUTO
  * - `server_flags` = IMAP-/SMTP-flags as a combination of @ref DC_LP flags, guessed if left out
- * - `socks5_enabled` = SOCKS5 enabled
- * - `socks5_host` = SOCKS5 proxy server host
- * - `socks5_port` = SOCKS5 proxy server port
- * - `socks5_user` = SOCKS5 proxy username
- * - `socks5_password` = SOCKS5 proxy password
+ * - `socks5_enabled` = Proxy enabled, named `socks5_enabled` for compatibility reason
```

Comment:
```suggestion
 * - `socks5_enabled` = "Proxy enabled", named `socks5_enabled` for compatibility reasons.
```

Reply:
you need to migrate the proxy anyways so I wonder why not start with a new "proxy_enable" setting